form.yml

```diff
@@ -0,0 +1,329 @@
+    $type: boolean
+    $default: true
+    $help: Enable the HA cluster configuration. In order to make this work, ASCS and ERS system must be installed together in parallel.
+    $optional: false
+
+  nfs_version:
+    $name: NFS version
+    $type: text
+    $default: nfs4
+    $help: Used to connect to the nfs share
+    $optional: true
+
+  nfs_options:
+    $name: NFS options
+    $type: text
+    $default: defaults
+    $help: Specify the options for nfs share
+    $optional: true
+
+  additional_dvds:
+    $name: Additional dvds
+    $type: edit-group
+    $help: Path to software folder or archives with additional SAP software
+    $minItems: 1
+    $prototype:
+        $name: Additional DVD
+        $type: text
+        $optional: false
+
+  saptune_solution:
+    $name: saptune solution to apply
+    $type: text
+    $default: NETWEAVER
+    $help: saptune solution to apply to all nodes
+    $optional: true
+
+  hana_checkbox:
+    $name: Add information about installed HANA database (mandatory for DB/PAS/AAS instances)
+    $type: boolean
+    $default: false
+    $help: Information about the already deployed HANA Database. It is mandatory if DB/PAS/AAS are to be installed
+  hana:
+    $name: Hana database information
+    $visibleIf: .hana_checkbox == true
+    $type: group
+    $help: Information about the already deployed HANA Database. Mandatory for DB/PAS/AAS instances
+    host:
+      $name: Host
+      $type: text
+    sid:
+      $name: SID
+      $type: text
+    instance:
+      $name: Instance
+      $type: text
+    password:
+      $name: Password
+      $type: password
+
+  schema_checkbox:
+    $name: Add schema information from DB installation
+    $type: boolean
+    $default: false
+    $help: Information about the Schema created during DB installation and used by PAS/AAS
+  schema:
+    $name: Schema
+    $visibleIf: .schema_checkbox == true
+    $type: group
+    $help: Information about the Schema created during DB installation and used by PAS/AAS
+    name:
+      $name: Schema name
+      $type: text
+    password:
+      $name: Password
+      $type: password
+
+  product_id:
+    $name: SAP product ID
+    $type: text
+    $help: The installed SAP product id. If some particular node has its own product_id that one will have the preference. The options are. NW750.HDB.ABAPHA, S4HANA1709.CORE.HDB.ABAPHA, S4HANA1809.CORE.HDB.ABAPHA, S4HANA1909.CORE.HDB.ABAPHA. Remove the 'HA' from the end for non HA environment (without ERS)
+    $optional: false
+
+  monitoring_enabled:
+    $name: Enable the host to be monitored by exporters
+    $type: boolean
+    $default: false
+    $help: Enable the node monitoring via exporters which will be installed and configured in all the nodes.
+
+  nodes:
+    $name: Nodes
+    $type: edit-group
+    $minItems: 1
+    $itemName: ""
+    $prototype:
+      host:
+        $name: Hostname to install Netweaver
+        $type: text
+        $optional: false
+      virtual_host:
```

Comment:
We have to make clear here, that this `ascs_virtual_host` must match with one of the entries we have added at the top in `virtual_hosts`